docs/supported_entities.md

```diff
@@ -76,6 +82,41 @@ For more information, refer to the [adding new recognizers documentation](analyz
 
 See [this documentation](analyzer/adding_recognizers.md) for instructions on how to add a new Recognizer for a new type of PII entity.
 
+## Complementing Presidio with Azure AI Language PII
+
+[Azure AI Language PII](https://learn.microsoft.com/en-us/azure/ai-services/language-service/personally-identifiable-information/overview)
+ is a cloud-based service that provides Natural Language Processing (NLP) features for detecting PII in text.
+
+A list of supported entities by Azure AI Language PII [can be found here](https://learn.microsoft.com/en-us/azure/ai-services/language-service/personally-identifiable-information/concepts/entity-categories).
+
+To add Azure AI language into Presidio:
+
+### Installation
```

Comment:
maybe link to the other markdown instead of repeating

Reply:
what markdown you're refering to?

Reply:
This one? https://github.com/microsoft/presidio/blob/omri/azure_language_nlp_engine/docs/samples/python/text_analytics/index.md

Reply:
Good call. I'll fix it.